lib/browser/app.js

```diff
@@ -59,6 +59,12 @@ app.controller('AppController', function($q, DriveScannerService, SelectionState
     this.selection.clear();
     this.writer.setProgress(0);
     this.scanner.start(2000);
+
+    this.scanner.scan().then(function (res) {
+      if(res.length === 1){
```

Comment:
Another very low priority one, but there is a space missing between the closing parenthesis and the open curly brace.